db/go/20210323175655_update_templates_for_pns.go

```diff
@@ -0,0 +1,80 @@
+package migration
+
+import (
+	"database/sql"
+	"github.com/pressly/goose"
+	"path/filepath"
+)
+
+func initialize20210323175655() {
+	if _, ok := initializedMigrations[20210323175655]; !ok {
+		goose.AddMigration(Up20210323175655, Down20210323175655)
+		initializedMigrations[20210323175655] = true
+	}
+}
+
+// Up20210323175655 update workflows to support new PNS mode
+func Up20210323175655(tx *sql.Tx) error {
+	// This code is executed when the migration is applied.
+	if err := updateWorkflowTemplateManifest(
+		filepath.Join("workflows", "hyperparameter-tuning", "20210323175655.yaml"),
+		hyperparameterTuningTemplateName,
```

Comment:
We actually need to delete this since it's in starter templates now - docs are updated to reflect this change.

Reply:
On it.

Reply:
Based on discussion with Rush, we are going to do a conditional delete in case someone has actually used this template.

If they have not, it will be deleted.